src/azul/service/elasticsearch_service.py

```diff
@@ -357,13 +368,21 @@ def _generate_paging_dict(self, es_response, pagination):
             'search_after_uid': search_after[1],
             'search_before': search_before[0],
             'search_before_uid': search_before[1],
+            'next': next_,
+            'previous': previous,
             'pages': pages,
             'sort': pagination['sort'],
             'order': pagination['order']
         }
 
         return page_field
 
+    def page_link(self, pagination, sort_facet, **kwargs) -> str:
```

Comment:
If you make this a nested function in _generate_paging_dict you can eliminate passing the `pagination` and `sort_facet`.